src/dso_api/dynamic_api/serializers.py

```diff
@@ -238,9 +238,11 @@ def to_representation(self, validated_data):
                 source_id = validated_data.id
                 through_tabel_filter_params = {source_field_name: source_id}
                 target_id_field = f"{related_mgr.target_field_name}_id"
-                through_tabel_items = related_mgr.through.objects.filter(
-                    **through_tabel_filter_params
-                ).values_list(target_id_field, flat=True)
+                through_tabel_items = (
+                    related_mgr.through.objects.filter(**through_tabel_filter_params)
+                    .order_by()
```

Comment:
Moet `order_by` niet tenminste één argument hebben?

Reply:
Het ontbreken van een argument betekent dat de ORDER BY clause weggelaten wordt. En dat is hier de bedoeling.